exercises/handling-get-requests/views/home.erb

```diff
@@ -10,10 +10,26 @@
 <% end %>
 
 <h3>Search!</h3>
-<p>Wish there was a search field...</p>
+<p>Wow look a form for search!</p>
+
+<form action="/search" method = "get">
+  <input type = "text" name = "terms" /> <br />
+  <input type= "submit" />
+</form>
 
 <h5>Cool Searches</h5>
 <p>Wish there was a list of recommended searches...</p>
 
+<ul>
+  <li><a href="http://localhost:9292/search?terms=smart">Cool search for "smart"</a></li>
```

Comment:
Be careful!  You don't want to hard-code this kind of hostname.  Your code will now only work on a computer which runs the code on `localhost` on port `9292`.

You can omit the hostname in a link tag and it will automatically link to the given path on the current host, e.g.,

```html
<a href="/search?terms=man">Cool search for "man"</a>
```